### ISSUE TITLE
[BUG] Installation fails because an unexpected error occurred: "ENOTDIR: not a directory {...}/ISSUE_TEMPLATE/BUG_REPORT.md'".

### ISSUE THREAD

**Issue by giuliogallerini** [google-map-react/google-map-react#830]
**Describe the bug 🐛**

During our CI dependencies installation, `google-map-react` fails to install because of the following error:

```bash
error An unexpected error occurred: "ENOTDIR: not a directory, lstat 'PATH_TO_NODE_MODULES/node_modules/google-map-react/.github/ISSUE_TEMPLATE/BUG_REPORT.md'".
```

**To Reproduce 🔍**

1. Run `yarn upgrade`
2. See the error in the console.

**Expected behavior 💭**

The installation flow works.

**Environment:**

 - OS: MacOSX
 - iTerm
 - Version 3.3.8

**Comment by creaumond** [comment]
Deleting the `node_modules` folder and then running `yarn install` resolved the problem for me.

**Comment by the-yadu** [comment]
This is happening on netlify build too which basically creates a new build every time.

**Comment by itsmichaeldiego** [comment]
Glad it worked!!!

**Comment by pedrosimao** [comment]
I have the same problem. Deleting `node_modules` won't help...

**Comment by giuliogallerini** [comment]
@itsmichaeldiego I am not sure why this issue was closed without a proper solution. There are 19 👍, which means the issue is still active. Please reopen it, I might send a PR for a fix.

As a side note (@creaumond): If you have an older version then `v1.1.6` in your `package.json` and you clear your node_modules, when you run `yarn install` that version is going to be installed. As soon as you run `yarn upgrade` it will fail again. Our CI (CircleCI) runs this cmd automatically and therefore it always happens during the installation.

## TEMP FIX 🎉 
As a temporary fix for our project, we set google-map-react to the latest working fixed version: `"google-map-react": "1.1.5",`

**Comment by beevelop** [comment]
Same here. We fixed the version to `1.1.5`. Every time we try to bump the dependency it fails with exception from above. This issue doesn't seem to be solved yet.

**Comment by creaumond** [comment]
@giuliogallerini  - I believe you are right.

It looks like in  `1.1.5` had a file named `.github/ISSUE_TEMPLATE `. `1.1.6` removes that file and adds `.github/ISSUE_TEMPLATE/BUG_REPORT.md`. 

`yarn install` breaks because it is trying to create the file `BUG_REPORT.md` inside of `ISSUE_TEMPLATE`, which is a file and not a folder. 

I was able to successfully install `1.1.6` by removing the package, running `yarn install`, adding it back @ `1.1.6` and running `yarn install` again. I'd imagine you could also just delete the folder `node_modules/google-map-react/.github` and your install would work then as well.

**Comment by nicoalonsop** [comment]
I had the same problem. I fix it running `rm node_modules/google-map-react/.github/ISSUE_TEMPLATE`. It is not necessary to remove all `node_modules`

**Comment by hinok** [comment]
The proper fix is defining `files` property in `package.json` and selecting only required files/directories like https://docs.npmjs.com/files/package.json#files

```json
"files": [
    "/dist",
    "/lib",
    "/utils"
],
```

Things like `.vscode`, `.github`, `test`, `develop`, `.flowconfig`, `.travis.yml`, `webpack.config.*.js` are probably not required to be packaged.

@istarkov @itsmichaeldiego right? I can create PR.

**Comment by itsmichaeldiego** [comment]
Hey everyone, I merged #835 from @giuliogallerini that fixes this issue, and its published in v1.1.7
